testutil/testpolylog/event.go

```diff
@@ -45,26 +45,50 @@ func (m *FnMethodSpy) Fn(event polylog.Event) {
 	m.Called(event)
 }
 
-// EventMethodsTest is a test case for expressing and exercising polylog.Event
+// EventMethodTestCase is a test case for expressing and exercising polylog.Event
 // methods in a concise way.
-type EventMethodsTest struct {
-	Msg                    string
-	MsgFmt                 string
-	MsgFmtArgs             []any
-	Key                    string
-	Value                  any
-	EventMethodName        string
+type EventMethodTestCase struct {
+	// Msg is the string to pass to polylog.Event#Msg(), which will be called
+	// after the event method under test. Usage of Msg is mutually exclusive
+	// with MsgFmt. If neither are provided, then polylog.Event#Send() is called
+	// after the event method instead.
+	Msg string
+
+	// MsgFmt is the format string to pass to polylog.Event#Msgf(), which will
+	// be called on the event returned from the event method under test. Usage
+	// of MsgFmt is mutually exclusive with Msg. If neither are provided, then
+	// polylog.Event#Send() is called after the event method instead.
+	MsgFmt string
+
+	// MsgFmtArgs are the args to pass to polylog.Event#Msgf(). It is an error
+	// to provide MsgFmtArgs without also providing MsgFmt or while providing
+	// Msg.
+	MsgFmtArgs []any
+
+	// Key is the key to pass to the event method under test. If EventMethodName
```

Comment:
Reminder to remove `If EventMethodName...` or not depending on the outcome of our other discussion.

We'd be able to avoid the comment here, below, in the implementation in exchange for verbosity in test case definition.